nncf/common/pruning/operations.py

```diff
@@ -265,9 +265,9 @@ def generate_output_masks(cls, node: NNCFNode, graph: NNCFGraph,
         :return: Filled input masks.
         """
         input_masks = get_input_masks(node, graph)
-        if not input_masks:
-            return None
```

Comment:
I believe that it could be not safe because `get_input_masks` could return empty list

Reply:
I changed this to cover `[None]` case. I updated `generate_input_mask` function to cover empty list.